package-lock.json

```diff
@@ -1322,6 +1322,31 @@
         "graceful-fs": "^4.1.3",
         "mkdirp": "^0.5.1",
         "rimraf": "^2.5.2"
+      },
```

Comment:
One last thing, it looks like this package-lock.json is left over from when new dependencies were added. Try rerunning npm install (should regenerate this file) and I think the result will be no change to this.

Reply:
Thanks, nice catch! I've re-run `npm install` and `package-lock` is up to date now.